python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -743,16 +738,15 @@ def run_transformer_int4_gpu_win(repo_id,
                                                 num_beams=num_beams)
                     torch.xpu.synchronize()
                     end = time.perf_counter()
-                    reserved_mem_list.append(torch.xpu.memory.memory_reserved()/(1024**3))
-                    gpu_peak_mem = max(reserved_mem_list) # always keep the peak gpu mem at current stage
                     output_ids = output_ids.cpu()
                     print("model generate cost: " + str(end - st))
                     output = tokenizer.batch_decode(output_ids)
                     print(output[0])
+                    torch.xpu.empty_cache()
```

Comment:
`torch.xpu.empty_cache()` Previously we have some tests and found that this may make first token slower on iGPU

Reply:
It's real, removed. But on A770, the empty_cache doesn't affect the performance.